dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -4,13 +4,13 @@
     "wrapup_video_id": null,
     "login_required": false,
     "properties": {
-      "lesson_extras_available": true,
-      "hideable_lessons": true
+      "hideable_lessons": true,
```

Comment:
double checking: are all the changes in this file intentional?

Reply:
ah no. I think this is mostly noise but I'll clean up this diff before merging